src/mono/Directory.Build.props

```diff
@@ -6,6 +6,7 @@
   
   <!-- Set default Platform -->
   <PropertyGroup>
+    <HostArch>$([System.Runtime.InteropServices.RuntimeInformation]::ProcessArchitecture.ToString().ToLowerInvariant)</HostArch>
```

Comment:
Does it matter that `ToLowerInvariant` is missing parenthesis?

Reply:
Oops

Reply:
@am11 however it works without them too

Reply:
Great, i was also thinking that it (might) just work in property functions. 👍